client/rest/root.go

```diff
@@ -79,12 +77,19 @@ func (rs *RestServer) Start(listenAddr string, maxOpen int, readTimeout, writeTi
 func ServeCommand(cdc *codec.Codec, registerRoutesFn func(*RestServer)) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "rest-server",
-		Short: "Start LCD (light-client daemon), a local REST server",
+		Short: "Start a local REST server",
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
 			rs := NewRestServer(cdc)
 
 			registerRoutesFn(rs)
-			rs.registerSwaggerUI()
+
+			// TODO:
+			//
+			// 1. Build swagger docs at runtime as to allow user/client to specify
+			// general API description.
+
+			// mount REST (Swagger) documentation
+			rs.Mux.PathPrefix("/swagger/").Handler(httpswagger.WrapHandler)
```

Comment:
So we're removing the swagger-ui endpoint then right?

Reply:
Yes, we removed the `/swagger-ui/` endpoint which mounted the static/manual documentation in favor of `/swagger/` which now mounts the automated docs.


Do you think we should revert the name change? `/swager-ui/` never made much sense to me. Maybe we can keep both endpoints and deprecate one over time.

Reply:
I don't know enough if anyone depends on it, @jackzampolin you know?

Reply:
Well, this is simply just the route. If our changelog stipulates the new route, which it will, it shouldn't be a problem.

Reply:
The front end team will need to update some routing on the website.